lib/ansible/modules/cloud/azure/azure_rm_managed_disk.py

```diff
@@ -79,8 +79,13 @@
         required: false
     disk_size_gb:
         description:
-            -Size in GB of the managed disk to be created. If 'create_option' is 'copy' then the value must be greater than or equal to the source's size.
-        required: true
+            - Size in GB of the managed disk to be created. If 'create_option' is 'copy' then the value must be greater than or equal to the source's size.
+        required: false
+    managed_by:
```

Comment:
Are there plans on adding more objects that can manage the disks? if it is only a VM then it may be better to have this be `vm_name` but happy to be convinced otherwise.

Reply:
Actually, a disk can be managed by a VM or VMSS. And the `managed_by` is a field coming from SDK's disk model. I think keep the `managed_by` is better since it is consistent with SDK.